Cargo.toml

```diff
@@ -1,14 +1,186 @@
 [workspace]
 members = [
-    "crates/*",
+    # EDR
+    "crates/edr_defaults",
+    "crates/edr_eth",
+    "crates/edr_napi",
+    "crates/edr_provider",
+    "crates/edr_solidity",
+    "crates/edr_test_utils",
+    "crates/tools",
+    # Foundry
+    "crates/foundry/cheatcodes",
+    "crates/foundry/cheatcodes/spec",
+    "crates/foundry/common",
+    "crates/foundry/config",
+    "crates/foundry/evm/core",
+    "crates/foundry/evm/coverage",
+    "crates/foundry/evm/evm",
+    "crates/foundry/evm/fuzz",
+    "crates/foundry/evm/traces",
+    "crates/foundry/forge",
+    "crates/foundry/macros",
+    "crates/foundry/test-utils",
 ]
 resolver = "2"
 
+[workspace.package]
+version = "0.3.8"
+edition = "2021"
+
 [profile.dev]
 rpath = true
+# Speed up compilation time for dev builds by reducing emitted debug info.
+# NOTE: Debuggers may provide less useful information with this setting.
+# Uncomment this section if you're using a debugger.
+debug = 1
 
+# Optimized release profile.
 [profile.release]
 rpath = true
+opt-level = 3
+debug = "line-tables-only"
+lto = "fat"
+strip = "debuginfo"
```

Comment:
Does this imply stripping of debug info? This would prohibit sampling profiling

Reply:
Good catch on this setting. I looked up the [default release params](https://doc.rust-lang.org/cargo/reference/profiles.html#release) and there it's `debug = false` actually, so the `debug` and `strip` flags are superfluous here and I've removed them.

Regarding sampling profiling, that needs full debug info, so we have to run it with an override currently as well: `CARGO_PROFILE_RELEASE_DEBUG=true cargo flamegraph ...`